lib/bridge/MemberListSyncer.js

```diff
@@ -247,10 +271,63 @@ function joinMatrixUsersToChannels(rooms, server, injectJoinFn) {
     return d.promise;
 }
 
-function leaveIrcUsersFromRooms(rooms, server) {
+MemberListSyncer.prototype.leaveIrcUsersFromRooms = function(rooms, server) {
+    log.info("leaveIrcUsersFromRooms");
```

Comment:
Add some more information here for it to be useful: perhaps the number of rooms and the server domain.